server.py

```diff
@@ -1176,8 +1209,8 @@ def put(self):
         user = get_user_from_request(request, dao)
         if not user:
             return 'Permission denied', 403
-        #if not is_user_admin_for_region(user, region='*'):
-        #    return 'Permission denied', 403
+        if not dao.get_is_superadmin(user.id):
```

Comment:
hmm if we have the user object, can't we just check if user.admin_level=='SUPER'? This should save a mongo lookup.